docs/en/deployment/helm.md

```diff
@@ -94,6 +94,38 @@ displayed_sidebar: "English"
     helm uninstall starrocks
     ```
 
-- Search Helm Chart maintained by StarRocks on Artifact Hub
+## More information
 
-  See  [kube-starrocks](https://artifacthub.io/packages/helm/kube-starrocks/kube-starrocks).
+- The address of the GitHub repository: [starrocks-kubernetes-operator and kube-starrocks Helm Chart](https://github.com/StarRocks/starrocks-kubernetes-operator).
+
+- The docs in the GitHub repository provide more information, for example:
+
+  - If you need to manage objects like the StarRocks cluster via the Kubernetes API, see [API reference](https://github.com/StarRocks/starrocks-kubernetes-operator/blob/main/doc/api.md).
+
+  - If you need to mount persistent volumes to FE and BE pods to store FE metadata and logs, as well as BE data and logs, see [Mount Persistent Volumes by Helm Chart](https://github.com/StarRocks/starrocks-kubernetes-operator/blob/main/doc/mount_persistent_volume_howto.md#2-mounting-persistent-volumes-by-helm-chart).
+
+    :::danger
+
+    If you don't mount persistent volumes, the StarRocks Operator will use emptyDir to store FE metadata and logs, as well as BE data and logs. When containers restart, data will be lost.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/b5641e4e-3b3d-403c-8c77-29e6aaaa0130)
-> If persistent volumes are not mounted